ods_ci/tests/Resources/Files/triton/triton_pytorch_rest_servingruntime.yaml

```diff
@@ -0,0 +1,54 @@
+apiVersion: serving.kserve.io/v1alpha1
```

Comment:
isn't possible to use the same triton runtime as https://github.com/red-hat-data-services/ods-ci/pull/1737/files ? I see small differences, are those necessary? both the runtime list onnx and pytorch as supported format